scripts/test/SANS/user_file/user_file_state_director_test.py

```diff
@@ -220,3 +217,6 @@ def test_stat_can_be_crated_from_valid_user_file_and_later_on_reset(self):
 
 if __name__ == "__main__":
     unittest.main()
+
```

Comment:
hmm, I am wondering if this has happened during a merge. I have had some issues with merges going silently wrong. I will remove it. Thanks for spotting this